File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/ImageGenerationInput.kt
package com.sdkwork.backend

data class ImageGenerationInput(
    val title: String? = null,
    val prompt: String? = null,
    val negativePrompt: String? = null,
    val referenceAssets: List<AssetMediaResource>? = null,
    val storyboard: VideoStoryboard? = null,
    val parameters: Map<String, Any>? = null,
    val n: Int? = null,
    val seed: Int? = null,
    val format: String? = null,
    val width: Int? = null,
    val height: Int? = null,
    val size: String? = null,
    val quality: String? = null,
    val style: String? = null,
    val cfgScale: Double? = null,
    val steps: Int? = null,
    val sampler: String? = null,
    val strength: Double? = null,
    val safetyChecker: Boolean? = null,
    val aspectRatio: String? = null,
    val referenceAssetCount: Int? = null
)

File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/image_generation_input.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class ImageGenerationInput:
    """图片生成输入参数"""
    title: str = None
    prompt: str = None
    negative_prompt: str = None
    reference_assets: List[AssetMediaResource] = None
    storyboard: VideoStoryboard = None
    parameters: Dict[str, Any] = None
    n: int = None
    seed: int = None
    format: str = None
    width: int = None
    height: int = None
    size: str = None
    quality: str = None
    style: str = None
    cfg_scale: float = None
    steps: int = None
    sampler: str = None
    strength: float = None
    safety_checker: bool = None
    aspect_ratio: str = None
    reference_asset_count: int = None
